Resources/Textures/Objects/Fun/spraycans.rsi/meta.json

```diff
@@ -70,6 +79,22 @@
     {
       "name": "equipped-BELT",
       "directions": 4
+    },
+    {
+      "name": "clown-inhand-right",
+      "directions": 4
+    },
+    {
+      "name": "clown-inhand-left",
+      "directions": 4
+    },
+    {
+      "name": "spray-inhand-right",
+      "directions": 4
+    },
+    {
+      "name": "spray-inhand-left",
+      "directions": 4
     }
   ]
 }
```

Comment:
I've also noticed that painted objects lose their stroke when hovering over the cursor.

Reply:
this is now fixed. Now paints the sprite layers rather than applying a post shader, which means objects keep their outline.